src/frontend/components/navbar/Navbar.css

```diff
@@ -0,0 +1,48 @@
+:root{
+    --light-sky-blue:rgb(197, 204, 211);
```

Comment:
Please use hex codes for consistency.

Reply:
since i m more comfortable with RGB through out in my projects i hv used RGB only.  but i ll do check n try to use hex for next projects